main.tf

```diff
@@ -449,26 +451,26 @@ resource "helm_release" "castai_kvisor" {
 
   set {
     name  = "controller.replicas"
-    value = var.install_security_agent == true ? 1 : 0
+    value = 1
```

Comment:
I would remove the replicas here, so that the default value of whatever the chart decides is used.

Reply:
Good point 👍